add udp section in manifest and fix chart directory structure (#65)
beclab/docs: docs/developer/develop/package/manifest.md

```
:::

## Ports
Specify exposed ports
:::info Example
```Yaml
ports:
- name: aaa          # Name of entrance that provide udp service
  host: udp          # Ingress name of entrance that provide udp service
  port: 8899         # Port of entrance that provide udp service
  protocol: udp      # udp/tcp
- name: bbb
  host: udp
  port: 8090
  protocol: tcp
```
:::

> Olares OS will generate a random exposed port between 33333 and 36789. These ports can be accessed through the app entrance domain with the port on the local network. For example: `84864c1f.local.your_olares_id.olares.com:33805`.

Reviewer: ```suggestion
Olares automatically assigns a random port (33333-36789) for your app. Access your app through the entrance domain and the port. For example: `84864c1f.local.your_olares_id.olares.com:33805`.
```
